parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -63,7 +63,7 @@ else
     FULL_INSTALL_REQUIRED=true
 fi
 
-if [[ $OS != $COREOS_OS_NAME ]] && [[ $FULL_INSTALL_REQUIRED ]]; then
+if [[ $OS == $UBUNTU_OS_NAME ]] && [ "$FULL_INSTALL_REQUIRED" = "true" ]; then
```

Comment:
`[ "$FULL_INSTALL_REQUIRED" = "true" ]` is inconsistent with all the other `[[ $FULL_INSTALL_REQUIRED ]]` usages.

Reply:
there are no other `[[ $FULL_INSTALL_REQUIRED ]]` usages

Reply:
only `if $FULL_INSTALL_REQUIRED; then`

Reply:
that's what broke it